src/interfaces/resource-interfaces/resource-interface.ts

```diff
@@ -51,7 +51,8 @@ export interface RegistrationResourceIF {
       certifyClause: string
       entityDisplay: string
     }
-  }
+  },
+  clickFileAndPay: boolean
```

Comment:
This field is added to toggle the status whether the button "File and Pay" is clicked.

Reply:
This variable is added here because this ticket is about firm registration only. If this approach applies to all filings, then this variable should be removed and I will use the original variable.